doc/source/abstractions/app-interface.rst

```diff
@@ -61,45 +61,49 @@ within this method.
 
 .. code:: python
 
-    def create_open_region(self, frequency="1GHz", boundary="Radiation",
-                           apply_infinite_gp=False, gp_axis="-z"):
-       """Create an open region on the active editor.
-
-       Parameters
-       ----------
-       frequency : str, optional
-           Frequency with units. The  default is ``"1GHz"``.
-       boundary : str, optional
-           Type of the boundary. The default is ``"Radiation"``.
-       apply_infinite_gp : bool, optional
-           Whether to apply an infinite ground plane. The default is ``False``.
-       gp_axis : str, optional
-           The default is ``"-z"``.
-
-       Returns
-       -------
-       bool
-           ``True`` when successful, ``False`` when failed.
-
-       Examples
-       --------
-       Create an open region in the active editor at 1 GHz.
-
-       >>> hfss.create_open_region(frequency="1GHz")
-        
-       """
-       vars = [
-           "NAME:Settings",
-           "OpFreq:=", frequency,
-           "Boundary:=", boundary,
-           "ApplyInfiniteGP:=", apply_infinite_gp
-       ]
-       if apply_infinite_gp:
-           vars.append("Direction:=")
-           vars.append(gp_axis)
-
-       self._omodelsetup.CreateOpenRegion(vars)
-       return True
+    def create_open_region(
+        self, frequency="1GHz", boundary="Radiation", apply_infinite_gp=False, gp_axis="-z"
+    ):
+        """Create an open region on the active editor.
+
+        Parameters
+        ----------
+        frequency : str, optional
+            Frequency with units. The  default is ``"1GHz"``.
+        boundary : str, optional
+            Type of the boundary. The default is ``"Radiation"``.
+        apply_infinite_gp : bool, optional
+            Whether to apply an infinite ground plane. The default is ``False``.
+        gp_axis : str, optional
+            The default is ``"-z"``.
```

Comment:
```suggestion
            Axis for the ground plane. The default is ``"-z"``.
```

Reply:
This parameter needed to have a description. Please review and correct as necessary.